### DISCUSSION TITLE
闲暇时间可以看看镜像构建时间 || In your free time, you can take a look at the image build time

### DISCUSSION THREAD

**Discussion by LiangWei88** [blinkospace/blinko#297] [General]
大佬开发的速度好快, 几十分钟一个版本都有, 但是上个版本都还没打包完, 下个版本就出了, 我老是在纠结等待下个 release, 看了一下, 每次的 build 都花一个小时多, 如果由空闲时间, 可以看一眼哪里造成这么久的打包时间

---
The development speed of the boss is so fast, a version is ready in dozens of minutes, but the previous version has not been packaged yet, and the next version will be released. I am always struggling to wait for the next release. After looking at it, every time The build takes more than an hour. If you have free time, you can take a look at what causes such long packaging time

**Comment by LiangWei88** [comment]
刚刚自己拉下来 build 了一下 600s+
github action 这么久说不定是网络原因加上用了国内镜像?

---
I just pulled it down and built it for 600s+
Maybe the reason why github action takes so long is because of the network and the use of domestic mirrors?

**Comment by blinko-space** [comment]
now the lasted version improve to about 30 minutes on github action.may be more faster in your computer
